src/styles/styles.js

```diff
@@ -169,6 +169,43 @@ const styles = {
         width: variables.componentSizeNormal,
     },
 
+    picker: {
+        inputIOS: {
+            fontSize: variables.fontSizeNormal,
+            paddingVertical: 12,
+            paddingHorizontal: 10,
+            borderRadius: variables.componentBorderRadius,
+            borderColor: themeColors.border,
+            color: 'black',
+            paddingRight: 30,
+        },
+        inputWeb: {
+            fontSize: variables.fontSizeNormal,
+            paddingVertical: 12,
+            paddingHorizontal: 10,
+            borderWidth: 1,
+            borderRadius: variables.componentBorderRadius,
+            borderColor: themeColors.border,
+            color: 'black',
+            paddingRight: 30,
+            appearance: 'none',
+        },
+        inputAndroid: {
+            fontSize: variables.fontSizeNormal,
+            paddingHorizontal: 10,
+            paddingVertical: 8,
+            borderWidth: 1,
+            borderRadius: variables.componentBorderRadius,
+            borderColor: themeColors.border,
+            color: 'black',
+            paddingRight: 30,
+        },
+        iconContainer: {
+            top: 12,
+            right: 12,
+        },
+    },
+
```

Comment:
I know this is a different structure to the other styled. I chose it because I have to use these names for the picker library. I thought it would be tidier and more self documenting if they were grouped to the picker since I don't think these styles need to be re-used anywhere.

Maybe I could flatten them out with prefixed names and re-name them before passing them to the component if you don't like this approach.

Reply:
Seems Ok to me. We are not likely going to re-use these outside of this context so having them together here seems fine for now.